src/App.tsx

```diff
@@ -37,84 +26,13 @@ import "./theme/variables.css";
 
 setupIonicReact();
 
-export const PATHS = {
-  HOME_PAGE: "/home",
-  TUTORIAL_PAGE: "/tutorial",
-  WELCOME_PAGE: "/welcome",
-};
-
-const queryClient = new QueryClient({
-  defaultOptions: {
-    mutations: {
-      networkMode: "online",
-    },
-    queries: {
-      networkMode: "online",
-      staleTime: 1000 * 20, // 20 seconds
-      gcTime: 1000 * 60 * 60 * 24 * 7, // 1 week
-      retry: 0,
-    },
-  },
-});
-addDefaultMutationFns(queryClient);
-
 const App: React.FC = () => {
-  const [persister, setPersister] = useState<Persister | null>(null);
-
-  useEffect(() => {
-    async function initPersister() {
-      const store = new Storage();
-      await store.create();
-      setPersister(
-        createAsyncStoragePersister({
-          storage: {
-            getItem: async (key) => store.get(key),
-            setItem: async (key, value) => store.set(key, value),
-            removeItem: async (key) => store.remove(key),
-          },
-        }),
-      );
-    }
-
-    if (persister == null) {
-      initPersister();
-    }
-  }, []);
-
-  if (persister == null) {
-    return null;
-  }
-
   return (
-    <PersistQueryClientProvider
-      client={queryClient}
-      persistOptions={{ persister: persister, maxAge: Infinity }}
-      onSuccess={() => {
-        queryClient.resumePausedMutations().then(() => {
-          queryClient.invalidateQueries();
-        });
-      }}
-    >
-      <IonApp>
-        <IonReactRouter>
-          <IonRouterOutlet>
-            <Route exact path={PATHS.WELCOME_PAGE}>
-              <WelcomePage />
-            </Route>
-            <Route exact path={PATHS.TUTORIAL_PAGE}>
-              <TutorialPage />
-            </Route>
-            <Route exact path={PATHS.HOME_PAGE}>
-              <Home />
-            </Route>
-            <Route exact path="/">
-              <Redirect to={PATHS.WELCOME_PAGE} />
-            </Route>
-          </IonRouterOutlet>
-        </IonReactRouter>
-      </IonApp>
-      <ReactQueryDevtools initialIsOpen={false} />
-    </PersistQueryClientProvider>
+    <StoreProvider>
+      <QueryClientProvider>
+        <Routes />
```

Comment:
I'd prefer the `Routes` component have a singular noun name: `Router` or `AppRouter`. In addition, I'd prefer that the `Routes` component's outermost `<IonApp>` element be here at the app level (so that the resulting outermost element of the `Routes` component is some kind of "router" thing).

Reply:
Makes sense to me. Thanks for the suggestion.